test/bigtest/tests/check-in-test.js

```diff
@@ -79,6 +79,26 @@ describe('CheckIn', () => {
     });
   });
 
+  describe('changing check-in date and time without loan', () => {
+    beforeEach(async function () {
+      this.server.post('/circulation/check-in-by-barcode', (_, request) => {
+        const body = JSON.parse(request.requestBody);
+        body.item = {};
+
+        return body;
+      });
+
+      await checkIn.clickChangeTime();
+      await checkIn.processDate.fillAndBlur('04/25/2018');
+      await checkIn.processTime.fillInput('11:12 PM');
```

Comment:
The value '11:12 PM' can be moved to the constant and it will be easier to support it in several places